app/views/groups/show.html.erb

```diff
@@ -1,2 +1,11 @@
 <%= render 'group_details' %>
-<%= link_to 'Delete', draw_group_path(@draw, @group), method: :delete %>
+<% if policy(@group).accept_request? %>
+  <h2>Membership Requests</h2>
+  <% @group.requests.each do |u| %>
+    <%= link_to "Accept #{u.full_name}", draw_group_accept_request_path(@draw, @group, params: { 'user_id' => u.id }), method: :put %>
+  <% end %>
+<% end %>
+<% if policy(@group).request_to_join? %>
+  <%= link_to 'Request To Join', draw_group_request_path(@draw, @group), method: :post %>
+<% end %>
+<%= link_to('Delete', draw_group_path(@draw, @group), method: :delete) if policy(@group).destroy? %>
```

Comment:
Another note, but this feels like it will be worth extracting into a partial for the footer where we can pass in the object in question as a local variable, since `destroy?` is always the relevant action across all the policies. I'll open an issue for this.